tests/test_core.py

```diff
@@ -2083,7 +2083,16 @@ def test_tcgetattr(self):
   def test_time(self):
     src = open(path_from_root('tests', 'time', 'src.cpp'), 'r').read()
     expected = open(path_from_root('tests', 'time', 'output.txt'), 'r').read()
-    self.do_run(src, expected);
+    try:
+      for tz in ['EST+05EDT', 'UTC+0']:
```

Comment:
Can we keep the current execution without TZ set, that is, that this PR only adds 2 more executions? (can do it before the loop, if so)

Reply:
Done